net/net.go

```diff
@@ -499,7 +509,7 @@ func (t *net) CreateRecord(ctx context.Context, id thread.ID, body format.Node)
 		return nil, err
 	}
 
-	log.Debugf("added record %s (thread=%s, log=%s)", rec.Cid().String(), id, lg.ID)
+	log.Debugf("added record %s (thread=%s, log=%s)", rec.Cid().String(), id.String(), lg.ID.String())
```

Comment:
I'm curious about this change since I've seen it in other places. 
The `%s` format already calls `.String()`: [example](https://play.golang.org/p/CFm00mmwrEM)

Do you have this as a lint check or similar?

Reply:
Oh yeah, the the usage is mixed so I was just going for consistency. I'll take a quick look and change what I can see in `net`.